CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Table of Contents
 
 ### Breaking Changes
 
+* Reports before version 1.7.0 cannot be loaded anymore.
```

Comment:
Maybe add: "Simply re-run your tests with the new recheck version to create them again."